lib/builder/webpack/client.js

```diff
@@ -15,6 +15,16 @@ export default class WebpackClientConfig extends WebpackBaseConfig {
     super(builder, { name: 'client', isServer: false })
   }
 
+  getFileName(...args) {
+    if (this.options.build.analyze) {
+      const key = args[0]
+      if (['app', 'chunk'].includes(key)) {
+        return '[name].js'
```

Comment:
What's the behaviour of making a production build with `--analyze` flag? Does it remove hashes? If yes I think it is a slightly breaking change for some users. (Not ME :D)

Reply:
It's not for production, just for `yarn build --analyze`.

If I want to watch bundle size and use `yarn build --analyze`, the file name in report is [name] should be more understandable, I suppose that nobody will use `yarn build --analyze` to generate production bundles, it's just for generate report.

Reply:
I think added this feature for a while. Not sure why and by was reverted 😆  Anyway I like it and just wanted to mention that fact. And also a really nice idea about the warning.

Reply:
Thanks for the reminding, I also remembered it's a legacy feature, so I marked pr as a bug fix. 😸